remove_en/remove.ftl

```diff
@@ -361,12 +362,17 @@ remove-modal-link-text = The following links are the web addresses where your in
 
 remove-modal-risk-title = What is Risk Level?
 
-# Data Removal Opt Out Modal
+# Data Removal Manual Modal
 
-remove-modal-optout-title = Manual Removal Instructions
-remove-modal-optout-text = If your request is blocked, or if for any reason you wish to attempt to manually remove your information,
+remove-modal-manual-title = Manual Removal Instructions
+remove-modal-manual-text = If your request is blocked, or if for any reason you wish to attempt to manually remove your information,
  you can use the contact email or website provided for this result to do so.
 
+ # Data Removal Optout Modal
+
+remove-modal-optout-title = Confirm Opt-Out of Data Removal Pilot
```

Comment:
nit: `Opt-Out` is hyphenated here (and L374 below), but not on L84 above.

Reply:
👍🏽